app/views/status_updates/show.html.slim

```diff
@@ -1,5 +1,8 @@
 = content_tag :h1, @status_update.title
 
+= link_to '<span>Atom Feed</span>'.html_safe, status_updates_show_path(status_update: @status_update, format: :atom),
```

Comment:
What's the purpose of the `span` here? If it's needed this is a bit of a hack, you can just as easily do the following:

```html
<a href="#{status_updates_show_path(....)">
  <span>Atom Feed</span>
</a>
```